db/migrate/20200823203902_create_accounts.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccounts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :accounts do |t|
+      t.string :financial_institution, null: false
+      t.string :description
+      t.decimal :initial_amount, null: false, default: 0.0
+      t.integer :account_type, null: false
```

Comment:
In this case, since the account type have limited possible values the enum solution seems to solve it.
Do you think that would be interesting to the user create new account types?